Bio/motifs/minimal.py

```diff
@@ -0,0 +1,162 @@
+# By Ariel Aptekmann 2017
+# Adapted from Bio.motifs 2008 by Bartek Wilczynski
+# Adapted from  Bio.MEME.Parser by Jason A. Hackney.  All rights reserved.
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+
+from __future__ import print_function
+
+from Bio.Alphabet import IUPAC
+from Bio import Seq
+from Bio import motifs
+import math
+
+def read(handle):
+    """Parses the text output of the MEME program into a meme.Record object.
+    Example:
+    >>> from Bio.motifs import meme
+    >>> with open("meme.output.txt") as f:
+    ...     record = meme.read(f)
+    >>> for motif in record:
+    ...     for instance in motif.instances:
+    ...         print(instance.motif_name, instance.sequence_name, instance.strand, instance.pvalue)
+    """
+    motif_number = 0	
+    record = Record()
+    __read_version(record, handle)
+    __read_alphabet(record, handle)
+    __read_background(record, handle)
+
+    while True:
+    	for line in handle:
+		if line.startswith('MOTIF'): 
+	    		break
+	else :
+		return record
+	name  = line.split()[1]
+	motif_number +=1
```

Comment:
Please include white space before and after the ``+=`` operator - this will be failing the flake8 style checks under TravisCI.